charts/kyverno/templates/background-controller/clusterrole.yaml

```diff
@@ -18,6 +18,16 @@ metadata:
   labels:
     {{- include "kyverno.background-controller.labels" . | nindent 4 }}
 rules:
+{{- if .Values.backgroundController.rbac.enableWildcardRBAC }}
+  - apiGroups:
+      - '*'
+    resources:
+      - '*'
+    verbs:
+      - get
+      - list
+      - watch
+{{- end }}
```

Comment:
Do we need to add more permissions here ?

Reply:
This is to maintain backward compatibility as they are part of the default installation prior to this change.